tests/plugins/test_pulp_pull.py

```diff
@@ -247,10 +247,8 @@ def test_pull_first_time(self, no_headers, broken_response, insecure, schema_ver
         # fall through to pulp_pull.
         plugin = PulpPullPlugin(tasker, workflow, insecure=insecure,
                                 timeout=0.1, retry_delay=0.25)
-        results, version = plugin.run()
+        version = plugin.run()
 
-        # Plugin return value is the new ID and schema
-        assert results == test_id
```

Comment:
We should be verifying that the image_id is updated in the workflow, shouldn't we?

Reply:
nm, I see it now